css/style.css

```diff
@@ -0,0 +1,83 @@
+* {
+    box-sizing: border-box;
+  }
+  body {
+    margin: 0;
+  }
+  .label{
+    width:100%;
+    display:block;
+  }
+  *{
+    box-sizing:border-box;
+  }
+  body{
+    margin:0;
+  }
+  .row{
+    display:flex;
+    justify-content:flex-start;
+    align-items:stretch;
+    flex-wrap:nowrap;
+    padding:10px;
+  }
+  .cell{
+    min-height:75px;
+    flex-grow:1;
+    flex-basis:100%;
+  }
+  .form-group{
+    padding:5px;
+  }
+  #iq53j{
+    color:white;
+    background-image:linear-gradient(#000000,#000000);
```

Comment:
I would suggest using a simple background-color and kindly remind you of using #000 instead of #000000